packages/dashboard/dashboard/index.py

```diff
@@ -1,44 +1,24 @@
-# Run this app with `python app.py` and
-# visit http://127.0.0.1:8050/ in your web browser.
-
+import os
 from dash import Dash, html, dcc
-import plotly.express as px
-import pandas as pd
+import dash
 from dotenv import load_dotenv
-import os
 
 load_dotenv()
 
-DEBUG_MODE = os.getenv("DEBUG").upper() == "TRUE"
+API_URL = os.getenv("API_URL")
 
-app = Dash(__name__)
-
-
-# assume you have a "long-form" data frame
-# see https://plotly.com/python/px-arguments/ for more options
-df = pd.DataFrame(
-    {
-        "Fruit": ["Apples", "Oranges", "Bananas", "Apples", "Oranges", "Bananas"],
-        "Amount": [4, 1, 2, 2, 4, 5],
-        "City": ["SF", "SF", "SF", "Montreal", "Montreal", "Montreal"],
-    }
-)
+app = Dash(__name__, use_pages=True)
 
-fig = px.bar(df, x="Fruit", y="Amount", color="City", barmode="group")
 
 app.layout = html.Div(
-    children=[
-        html.H1(children="Hello Dash"),
-        html.Div(
-            children="""
-        Dash: A web application framework for your data.
-    """
-        ),
-        dcc.Graph(id="example-graph", figure=fig),
+    [
+        html.H1("Jackal Analytics Dashboard"),
```

Comment:
No -- we can't do things this way. I don't like the idea of mixing different networks across different sections of the repo. We need to either make new repos or use separate branches.

Reply:
Yea didn't think about that. Going to make it an environment variable for now then